EXTRACTORS/DATAFRAME/EXTRACT_COLUMNS/EXTRACT_COLUMNS.py

```diff
@@ -7,15 +7,21 @@ def EXTRACT_COLUMNS(default: DataFrame | Matrix, columns: Array) -> DataFrame:
     """
     The EXTRACT_COLUMNS node takes an input dataframe/matrix and returns a dataframe/matrix with only the specified columns.
 
+    Inputs
+    ------
+    default : DataFrame|Matrix
+        Input to use as the table for column extraction
+
     Parameters
     ----------
     columns : list of str or list of int
-        The columns to extract from the input dataframe.
+        The columns to extract from the input dataframe
 
     Returns
     -------
-    DataFrame | Matrix
-        The dataframe or matrix with only the specified columns.
+    DataFrame|Matrix
+        type 'dataframe' with only the specified columns if the input was dataframe
```

Comment:
should be DataFrame and Matrix

Reply:
I've made it so that if the input is DataFrame output should be DataFrame and etc. Shouldn't it either DataFrame or Matrix?

Reply:
oh, I mean the casing, dataframe -> DataFrame, matrix -> Matrix

Reply:
oh aha okok mb! They've been changed!